mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -46,7 +46,9 @@ public CassandraAttachmentModule() {
                     .addPartitionKey(CassandraAttachmentTable.ID, text())
                     .addColumn(CassandraAttachmentTable.PAYLOAD, blob())
                     .addColumn(CassandraAttachmentTable.TYPE, text())
-                    .addColumn(CassandraAttachmentTable.SIZE, bigint())));
+                    .addColumn(CassandraAttachmentTable.SIZE, bigint())
+                    .withOptions()
+                    .comment("Stores attachment for fast JMAP retrieval")));
```

Comment:
Do you think it's specific to JMAP ? I think we could make use of it with IMAP, too, it's not the case ?
Why do you use "store" instead of "holds" like for other tables ?